interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -164,9 +165,9 @@
 
         /* Initialise Datetime Picker */
         $('.datetimepicker').datetimepicker({
-            <?php $datetimepicker_timepicker = true; ?>
-            <?php $datetimepicker_showseconds = true; ?>
-            <?php $datetimepicker_formatInput = false; ?>
+            <?php $datetimepicker_timepicker = false; ?>
+            <?php $datetimepicker_showseconds = false; ?>
+            <?php $datetimepicker_formatInput = (($_SESSION['language_direction'] == 'rtl') ? true : false); ?>
```

Comment:
Should be able to just make this true always (and then can test that it works for all 3 date_display_format global selections).